Sources/SotoCore/AWSClient+Paginate.swift

```diff
@@ -15,255 +15,81 @@
 import Logging
 import NIOCore
 
-// MARK: Pagination
-
 /// Protocol for all AWSShapes that can be paginated.
 /// Adds an initialiser that does a copy but inserts a new integer based pagination token
 public protocol AWSPaginateToken: AWSShape {
     associatedtype Token
     func usingPaginationToken(_ token: Token) -> Self
 }
 
+@available(macOS 10.15, iOS 13.0, tvOS 13.0, watchOS 6.0, *)
```

Comment:
This might need tidying up depending on the order of merges

Reply:
fixed